tests/testthat/test-case-variables.R

```diff
@@ -35,8 +35,8 @@ with_mock_crunch({
                                           missing="nope")),
                      "a case's missing must be a logical")
         expect_error(ensureValidCase(list(not_right="not")),
-                     paste("each case must have at most an id, name, expression,",
-                     "numeric_value, and missing element. The errant",
+                     paste0("each case must have at most an id, name, expression, ",
```

Comment:
Why all of these `paste` to `paste0` changes?

Reply:
I feel like we more frequently used `paste0` in tests (I just searched and it turns out it's almost exactly half and half). I'll remove the changes here.